mmdet/models/utils/activations.py

```diff
@@ -0,0 +1,23 @@
+import torch
+from torch import nn
+
+
+class MemoryEfficientSwishImplementation(torch.autograd.Function):
```

Comment:
It is recommended to change to the registrar mode, so that other places can also be replaced by configuration.

Reply:
We should add it to the `ACTIVATION_LAYERS` in MMCV. See example here https://github.com/open-mmlab/mmcv/blob/c47c9196d067a0900b7b8987a8e82768edab2fff/mmcv/cnn/bricks/activation.py